packages/mongodb-rag-core/src/contentStore/EmbeddedContent.ts

```diff
@@ -75,6 +75,7 @@ export type EmbeddedContentStore = VectorStore<EmbeddedContent> & {
   deleteEmbeddedContent(args: {
     page?: Page;
     dataSources?: string[];
+    inverse?: boolean;
```

Comment:
```suggestion
    inverseDataSources?: boolean;
```


also add a doc comment pls